.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Perhaps this file shouldn't be included?

Reply:
Thanks for the review, it's definitely not necessary although it is helpful to have around to debug in VSCode.  However not everybody uses it of course so can remove it.